app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -55,11 +50,15 @@ import com.google.android.libraries.places.widget.model.AutocompleteActivityMode
 import com.google.android.material.bottomsheet.BottomSheetBehavior
 import com.google.android.material.navigation.NavigationView
 import com.google.maps.android.PolyUtil
-import kotlinx.android.synthetic.main.bottom_sheet_layout.bottom_sheet
+import kotlinx.android.synthetic.main.bottom_sheet_layout.*
 import org.json.JSONObject
 import java.io.IOException
 import java.util.Locale
-import kotlin.system.exitProcess
+import android.widget.Toast
+import com.android.volley.Response
+import kotlinx.android.synthetic.main.bottom_sheet_layout.*
```

Comment:
we're not using this and the next dependency, so let's remove them

Reply:
Will do so :)